src/tests/data/block/kuznyechik.vec

```diff
@@ -0,0 +1,7 @@
+[Kuznyechik]
+
+# RFC 7801
+
+Key = 8899AABBCCDDEEFF0011223344556677FEDCBA98765432100123456789ABCDEF
+In = 1122334455667700FFEEDDCCBBAA9988
+Out = 7F679D90BEBC24305A468D42B9D4EDCD
```

Comment:
I would definitely want more than a single test for this. I would imagine at most 10% of the `T` table is used by this one particular vector.

If no tests are available from any other official sources (in which case ?????) then create random ones using another implementation.

Reply:
Created 64 random key/plain-text pairs and encrypted them with the 8-bit (slow) implementation available at https://github.com/mjosaarinen/kuznechik

Added the resulting vectors:

Kuznyechik ran 1235 tests in 2.73 msec all ok